src/test/java/com/hillel/items_exchange/util/UserDtoCreatingUtil.java

```diff
@@ -111,4 +113,46 @@ public static UserDto getBuild(String username, String email, Boolean online, St
                 .phones(phones)
                 .build();
     }
+
+    public static UserChangePasswordDto createUserChangePasswordDtoWithCorrectData() {
+
+        return createUserChangePasswordDto(CORRECT_OLD_PASSWORD,
+                NEW_PASSWORD,
+                NEW_PASSWORD);
+    }
+
+    public static UserChangePasswordDto createUserChangePasswordDtoWithWrongOldPassword() {
+
+        return createUserChangePasswordDto(WRONG_OLD_PASSWORD,
+                NEW_PASSWORD,
+                NEW_PASSWORD);
+    }
+
+    public static UserChangePasswordDto createUserChangePasswordDtoWithWrongPasswordConfirmation() {
+
+        return createUserChangePasswordDto(CORRECT_OLD_PASSWORD,
+                NEW_PASSWORD,
+                WRONG_NEW_PASSWORD_CONFIRMATION);
+    }
+
+    public static UserChangeEmailDto createUserChangeEmailDtoWithCorrectData() {
+
+        return createUserChangeEmailDto(NEW_VALID_EMAIL, NEW_VALID_EMAIL);
+    }
+
+    public static UserChangeEmailDto createUserChangeEmailDtoWithWrongEmailConfirmation() {
+
+        return createUserChangeEmailDto(NEW_VALID_EMAIL, NEW_INVALID_DUPLICATE_EMAIL);
```

Comment:
The same idea as I provided into the comment above.

Reply:
done